Source/Core/Core/HLE/HLE.cpp

```diff
@@ -185,13 +185,12 @@ bool IsEnabled(int flags)
 
 u32 UnPatch(const std::string& patch_name)
 {
-  auto* patch = std::find_if(std::begin(OSPatches), std::end(OSPatches), [&](const SPatch& patch) {
-    return patch_name == patch.m_szPatchName;
-  });
+  auto* patch = std::find_if(std::begin(OSPatches), std::end(OSPatches),
```

Comment:
tbh, I'd still go with find_if, even in the scenario of overhead in non-critical code, considering it's declarative and immediately says what you're doing based off the name.